src/toggle-switch/toggle-switch.scss

```diff
@@ -2,194 +2,212 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
 
+$toggle-switch-handle: $iui-icons-default;
+
 @mixin iui-toggle-switch {
   @include iui-reset;
   display: flex;
   align-items: center;
+  gap: $iui-s;
+  align-items: center;
   font-size: $iui-font-size;
   width: fit-content;
-  padding: ($iui-baseline/2) 0;
   user-select: none;
   cursor: pointer;
   position: relative;
+  isolation: isolate;
   @include themed {
     color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-2));
   }
 
-  > input {
-    width: 0;
-    height: 0;
+  // Disabled
+  &.iui-disabled {
+    cursor: not-allowed;
+    @include themed {
+      color: t(iui-text-color-muted);
+    }
+  }
+
+  // IE solution
+  // For labels on the right
+  > .iui-label {
+    margin-right: $iui-s;
+    @supports (gap: $iui-s) {
+      margin-right: 0;
+    }
+  }
+
+  // For labels on the left
+  .iui-toggle ~ .iui-label {
+    margin-left: $iui-s;
+    margin-right: 0;
+    @supports (gap: $iui-s) {
+      margin-left: 0;
+    }
+  }
+}
+
+@mixin iui-toggle {
+  display: grid;
+  grid-template: 'toggle';
+  align-items: center;
+
+  > input[type='checkbox'] {
```

Comment:
Do we really need to have attribute in selector because it is making specificity higher?

Reply:
I have removed it because it increases specificity.